client/src/app/management/components/committee-list/committee-list.component.ts

```diff
@@ -127,4 +141,12 @@ export class CommitteeListComponent extends BaseListViewComponent<ViewCommittee>
             fieldset: []
         };
     }
+
+    private canSeeCommittee(committee: ViewCommittee): boolean {
```

Comment:
Maybe you mean `canAccess`? This should be done by the autoupdate/restricter service -> You should only get a list of committees you can see. If you want to filter for access (similar to meetings), use the `canAccess` feature in the basemodel.

Reply:
I've understand this in another way -> but then, you have right.